forum/index.php

```diff
@@ -1,38 +1,41 @@
 <?php
 /*
-	Question2Answer by Gideon Greenspan and contributors
-	http://www.question2answer.org/
+    Question2Answer by Gideon Greenspan and contributors

```

Comment:
nie zmieniaj teraz formatowania - to spowoduje znów mi masę problemów przy debugowaniu (aczkolwiek widzę, że to zwykły index.php)

Reply:
fakt, mogłem tego nie robić, ale prawdę mówiąc nawet nie pomyślałem - odruchowo przeczyściłem cały plik zmieniając